assets/js/hooks/cell_editor/live_editor.js

```diff
@@ -566,8 +577,83 @@ class LiveEditor {
       );
     });
   }
+
+  clearDoctestDecorations() {
+    this._doctestDecorations.decorationCollection.clear()
+    this._doctestDecorations.deltaDecorations = []
+  }
+
+  _createDoctestDecoration(line, className) {
+    const lineNumber = Number(line)
```

Comment:
Isn't `line` always a number already?

Reply:
yes! previously I was sending `line` as an object key and it would arrive as a string. Will correct this.